src/interface/src/app/home/plan-table/plan-table.component.ts

```diff
@@ -40,31 +40,33 @@ export class PlanTableComponent implements OnInit {
   constructor(
     private dialog: MatDialog,
     private planService: PlanService,
-    private authService: AuthService,
     private router: Router,
     private snackbar: MatSnackBar
   ) {}
 
   ngOnInit(): void {
     this.getPlansFromService();
-    this.authService.isLoggedIn$.subscribe((_) => {
```

Comment:
I'm curious (for my own edification, not really as part of this review) what your reason is for removing this, because I also wasn't sure that is was doing us any good here.

Reply:
I think this was a remnant of the past that we never cleaned up. This component used to hold both logged in and logged out states. Now this is handled by `homeComponent`, and this component only makes sense if you are logged in.